service_storagefacility/src/it/scala/no/uio/musit/microservice/storagefacility/resource/KdReportResourceIntegrationSpec.scala

```diff
@@ -0,0 +1,84 @@
+package no.uio.musit.microservice.storagefacility.resource
+
+
+import no.uio.musit.microservice.storagefacility.domain.storage.StorageType._
+import no.uio.musit.microservice.storagefacility.domain.storage._
+import no.uio.musit.microservice.storagefacility.test.StorageNodeJsonGenerator._
+import no.uio.musit.microservice.storagefacility.test._
+import no.uio.musit.test.MusitSpecWithServerPerSuite
+import org.scalatest.time.{Millis, Seconds, Span}
+import play.api.http.Status
+import play.api.libs.json._
+import play.api.libs.ws.WSResponse
+
+
+/**
```

Comment:
FIXME
These "Created by..." comments aren't really useful. As a general rule it's better to leave them out, since a file typically will evolve so much over time that who originally wrote it doesn't really matter anymore. The author history of the file is in git anyways.

In your case, it's inserted by default by the IDE...so we should sit down and fix that so you don't have to think about it anymore :-)